.github/workflows/ci.yml

```diff
@@ -10,15 +10,18 @@ on:
 jobs:
     Test:
         runs-on: ${{ matrix.os }}
+        permissions:
+            id-token: write
```

Comment:
This workflow doesn't publish, so `id-token: write` is not needed in this workflow file. Same on line 34.